locales/en-US/messages.properties

```diff
@@ -30,9 +30,9 @@ credit_card=Credit / debit card
 submitting=Submitting…
 
 # privacy policy variants
-privacy_policy_var_a=We care about privacy and about helping you make informed choices. We want you to understand that your payment details will be processed by <a href="https://stripe.com/privacy/">Stripe</a> (for credit/debit cards) or <a href="https://www.paypal.com/webapps/mpp/ua/privacy-full">Paypal</a>, and a record of your donation will be stored by Mozilla.
-privacy_policy_var_b=Your payment details will be processed by <a href="https://stripe.com/privacy/">Stripe</a> (for credit/debit cards) or <a href="https://www.paypal.com/us/webapps/mpp/ua/privacy-full">Paypal</a>, and a record of your donation will be stored by Mozilla.
-privacy_policy_var_c=By making a donation, you acknowledge that your payment details will be processed by <a href="https://stripe.com/privacy/">Stripe</a> (for credit/debit cards) or by <a href="https://www.paypal.com/webapps/mpp/ua/privacy-full">Paypal</a> in accordance with their privacy policies, and a record of your donation will be stored by <a href="https://www.mozilla.org/privacy/">Mozilla</a>.
+privacy_policy_var_a=We care about privacy and about helping you make informed choices. We want you to understand that your payment details will be processed by <a href="https://stripe.com/privacy/">Stripe</a> (for credit/debit cards and SEPA Direct Debit) or <a href="https://www.paypal.com/webapps/mpp/ua/privacy-full">Paypal</a>, and a record of your donation will be stored by Mozilla.
```

Comment:
this seems like a significant change. You'll need to rename these keys and update them in code as well.

@TheoChevalier is that right?

Reply:
Absolutely, new keys are required for those

Reply:
I'll update the PR, how could I have forgotten!